runner/runner.py

```diff
@@ -64,6 +78,7 @@ def runner_arguments():
     parser.add_argument("BatchAccountSub", help="The batch account sub ")
     parser.add_argument("StorageAccountName", help="Storage name ")
     parser.add_argument("StorageAccountKey", help="storage key")
+    parser.add_argument("KeyVaultUrl", help="Azure Key vault to fetch secrets from, service principal must have access")
     parser.add_argument("ServicePrincipalCredentialsClientID", help="Service Principal id")
```

Comment:
if you make it an argument like so you'll always need to always add this to your command line. I would recommend making this optional since you can use this runner without the corona renderer and this arg wouldn't be needed.

Reply:
Can do, while only used for Corona now, keyVault params are generally available to other templates which might want to use the capability